modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -23,11 +23,12 @@ export interface ResizeHandler {
 }
 
 const barResizerPrefix = 'bar-';
+const isResizable = (table: SugarElement<HTMLTableElement>, elm: SugarElement<Element>) => Arr.forall([ table, elm ], (el) => Attribute.get(el, 'data-mce-resize') !== 'false');
```

Comment:
Do we need to pass both the `table` and `elm` here now? As given your other changes we should only need to pass the element, as if the table isn't resizeable it won't attempt to call `isResizable` on the cells/rows. As such, I'd suggest just passing the `elm` now.

Reply:
Just to add, since I know I suggested passing `table` that was because I hadn't considered we can just call `isResizable` on the table first and then do nothing if it's false.

Reply:
Yep, I agree that that table no longer needs to be passed in so have removed it from the callback.